addons/website_payment/static/tests/tours/donation.js

```diff
@@ -0,0 +1,122 @@
+odoo.define("website_payment.tour.donation_snippet_edition", function(require) {
+    "use strict";
+
+    const tour = require("web_tour.tour");
+    const wTourUtils = require("website.tour_utils");
+
+    tour.register("donation_snippet_edition", {
+        test: true,
+        url: "/?enable_editor=1"
+    }, [
+        wTourUtils.dragNDrop({
+            id: "s_donation",
+            name: "Donation"
+        }),
+        ...wTourUtils.clickOnSave(),
+        // -- Testing the minimum amount --
+        {
+            content: "Enter a negative custom amount, testing the minimum amount",
+            trigger: "#s_donation_amount_input",
+            run: "text -5"
+        },
+        {
+            content: "Donate with custom amount set",
+            trigger: ".s_donation_donate_btn",
+            run: "click"
+        },
+        {
+            content: "Check if alert-danger element exists",
+            trigger: "p.alert-danger"
+        },
+        // -- End of testing the minimum amount --
+        {
+            content: "Enter a custom amount",
+            trigger: "#s_donation_amount_input",
+            run: "text 55"
+        },
+        {
+            content: "Donate with custom amount set",
+            trigger: ".s_donation_donate_btn",
+            run: "click"
+        },
+        {
+            content: "Check if custom amount radio input is selected and value is 55",
+            trigger: "input#other_amount",
+            run: function(action) {
+                const input = action.tip_widget.$anchor[0];
+                const inputValue = input.nextElementSibling.children[0].value;
+                if (!input.checked || inputValue !== "55.0") {
+                    console.error("The custom amount radio input is not selected or the value is not 55.");
+                }
+            }
```

Comment:
We usually do not write custom `run` methods for such checks, you can have a step that looks for `[value="55"]` on the input, and a step that verifies `:checked` on the radio button.

To avoid that the matched element gets clicked, you can specify an empty run method:
```js
run: () => {}, // This is a check
```